test/integration/provider/openstack.go

```diff
@@ -76,7 +77,7 @@ func getOrphanedNICs(machineclass *v1alpha1.MachineClass, factory *client.Factor
 	orphans := []string{}
 	for _, port := range ports {
 		if err := network.DeletePort(port.ID); err != nil {
-			orphans = append(orphans, port.Name)
+			orphans = append(orphans, port.ID)
```

Comment:
we don't want to delete the resources in the `getOrpahnedInstances, getOrphanedDisks, getOrphanedNics` function, but just list and return them.

Reply:
like this https://github.com/gardener/machine-controller-manager-provider-aws/blob/1fe8064940be21d1df564002525ea8677a13e776/test/integration/provider/aws.go#L158-L160